coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -115,3 +140,23 @@ def get_ignore_comment(self, origin, language):
                         language=language))
                 ignore_comment = None
         return ignore_comment
+
+    def read_ignore_comments(self, original_line, comment_delimiter):
+        """
+        Read Ignore Comment(s) already present in the line and
+        accordingly return a list of bears previously ignored.
+        """
+        if comment_delimiter not in original_line:
+            return
+
+        ignore_regex = re.compile(re.escape(comment_delimiter)
+                                  + r'\sIgnore(\s\S+)+')
```

Comment:
this literal `Ignore` is problematic, as coala supports a variety of 'ignore' strings.

e.g. `start Ignoring foo`.

See `tests/processes/ProcessingTest.py` `test_yield_ignore_ranges`

Reply:
This functionality is for the ``Add Ignore Comment`` that coala adds itself, not for the Ignore statements we write. If we choose ``Add Ignore Comment`` option, coala adds ``somedelimiter + Ignore thisBear``

Reply:
This PR is to fix it so it can *amend* existing Ignore comments correctly.
The existing Ignore comments could have been written by a human, or by coala's Ignore feature.  It doesnt matter.

Reply:
Yes, the existing ignore comments can be written by humans or by coala but the amending ignore statement is written by coala only. And coala writes ignore comments for each line separately whereas ``start ignoring foo`` is for multiline ignores. Right?